data/helpers.d/utils

```diff
@@ -257,7 +257,15 @@ ynh_setup_source () {
 # | arg: ...         - (Optionnal) More POST keys and values
 ynh_local_curl () {
 	# Define url of page to curl
-	local full_page_url=https://localhost$path_url$1
+	path_url=$(ynh_normalize_url_path $path_url)
```

Comment:
path_url shouldn't be modified here, it should already have been normalized earlier in the install script.

Reply:
Ok, I'll remove this line